src/example/physics.cpp

```diff
@@ -1,26 +1,40 @@
-#include <chrono>
-#include <iostream>
-#include <thread>
+#include <crepe/ComponentManager.h>
+#include <crepe/system/RenderSystem.h>
+#include <crepe/system/CollisionSystem.h>
+#include <crepe/api/AssetManager.h>
+#include <crepe/system/PhysicsSystem.h>
+#include <crepe/api/loopManager.h>
 
 #include <crepe/Component.h>
-#include <crepe/ComponentManager.h>
-#include <crepe/api/Force.h>
 #include <crepe/api/GameObject.h>
 #include <crepe/api/Rigidbody.h>
 #include <crepe/api/Transform.h>
-#include <crepe/system/PhysicsSystem.h>
+#include <crepe/api/BoxCollider.h>
+#include <crepe/api/CircleCollider.h>
+#include <crepe/api/Sprite.h>
+#include <crepe/api/Texture.h>
+#include <crepe/api/Color.h>
+
+#include <memory>
 
 using namespace crepe;
 using namespace std;
 
 int main(int argc, char * argv[]) {
-	PhysicsSystem physics_system;
+	LoopManager game_loop;
 	GameObject * game_object[2];
```

Comment:
Dit is niet RAII.

Reply:
Je doet niks met de array (je gebruikt alleen game_object[0]). Waarom laat je het een array en verander je het niet gewoon naar een simpele variabele?